freeipa/src/main/java/com/sequenceiq/freeipa/sync/StackStatusCheckerJob.java

```diff
@@ -189,6 +197,7 @@ private void setStatusIfNotTheSame(InstanceMetaData instanceMetaData, InstanceSt
         if (instanceMetaData.getInstanceStatus() != newStatus) {
             if (updateStatus) {
                 instanceMetaData.setInstanceStatus(newStatus);
+                instanceMetaDataService.save(instanceMetaData);
```

Comment:
I think it might be saved via stack but I'm not sure. If you seen a missing status maybe it's best to save it here, but please test around with a few changes, even with HA when only one instance's status change to see if there is no issue. thanks

Reply:
Yes. My mistake. You are right. it is already saved via stack by cascading. The original code is working as it is. I confirmed in database and by debugging mode. I removed my change and kept it as before.